commands/src/config.rs

```diff
@@ -79,6 +79,33 @@ mod set {
     }
 }
 
+mod delete {
+    use crate::descriptions::config as descriptions;
+    use clap::{App, Arg};
+    use cto_ai::sdk::delete_config;
+
+    static KEY: &str = "key";
+
+    pub fn init_cli_command<'a, 'b>() -> App<'a, 'b> {
+        App::new("delete")
+            .about(descriptions::DELETE)
+            .arg(
+                Arg::with_name(KEY)
+                    .short("k")
+                    .long(KEY)
+                    .alias("name")
```

Comment:
I realized after looking at some of the other code that we're doing `sdk configs get <KEY>`, so we probably want to make this `sdk configs delete <KEY>`; that will involve replacing these three lines with `.index(1)`

Reply:
Addressed, thank you! https://github.com/cto-ai/sdk-bash/pull/3/commits/6a91629dfdfe4fce5421451970b79a7ea52951a7